doc/sphinx/source/ops/parameter.rst

```diff
@@ -0,0 +1,51 @@
+.. parameter.rst
+
+#########
+Parameter
+#########
+
+.. code-block: cpp
+
+   Paramerer // A function parameter.
+
+Description
+===========
+
+Parameters are nodes that represent the arguments that will be passed to user-defined functions.  Function creation requires a sequence of parameters.  Basic graph operations do not need parameters attached to a function.
```

Comment:
> Basic graph operations do not need parameters attached to a function.

Not sure what this means. (Is this just a holdover from our existing docs?)

Reply:
Holdover that is no longer with us.